lib/_http_parser.js

```diff
@@ -0,0 +1,890 @@
+/*
```

Comment:
Might be best to make this an internal module, a la [freelist](https://github.com/iojs/io.js/blob/master/lib/internal/freelist.js).